build.sbt

```diff
@@ -577,7 +577,8 @@ lazy val kernelApi = (project in file("kernel/kernel-api"))
       "com.fasterxml.jackson.core" % "jackson-core" % "2.13.5",
       "com.fasterxml.jackson.core" % "jackson-annotations" % "2.13.5",
       "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8" % "2.13.5",
-
+      "org.apache.iceberg" % "iceberg-api" % "1.6.1",
+      "org.apache.iceberg" % "iceberg-core" % "1.6.1",
```

Comment:
Can't use 1.7.1 at the moment because Delta only builds with JDK 8 at the moment as far as I can tell, with the exception of Spark master which is using JDk 17  (spark 4.0 requires JDK 17+)